src/app/players/pdf/pdf.component.html

```diff
@@ -27,6 +27,11 @@
             </section>
           </div>
       </div>
-  </div>
+    </div>
+
   </div>
 </div>
+
+<app-sn-popup [ShowsharePopup]="ShowsharePopup"></app-sn-popup>
```

Comment:
Share should be at app level rather than each player component level.

Reply:
I will send the separate PR Test cases and this base player component changes